{"content": {"suites": {"0": {"status": {"total_pass": 0, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 1, "total_error": 0}, "tests": {"0": {"status": "FAIL", "message": "   socket.gaierror: [Errno -3] Temporary failure in name resolution\n       urllib3.exceptions.NameResolutionError: <urllib3.connection.HTTPSConnection object at 0x72ea07e45330>: Failed to resolve 'googlechromelabs.github.io' ([Errno -3] Temporary failure in name resolution)\n       urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='googlechromelabs.github.io', port=443): Max retries exceeded with url: /chrome-for-testing/latest-patch-versions-per-build.json (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x72ea07e45330>: Failed to resolve 'googlechromelabs.github.io' ([Errno -3] Temporary failure in name resolution)\"))\n       requests.exceptions.ConnectionError: HTTPSConnectionPool(host='googlechromelabs.github.io', port=443): Max retries exceeded with url: /chrome-for-testing/latest-patch-versions-per-build.json (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x72ea07e45330>: Failed to resolve 'googlechromelabs.github.io' ([Errno -3] Temporary failure in name resolution)\"))\n       requests.exceptions.ConnectionError: Could not reach host. Are you offline?\n", "test_name": "test_packet_publishing", "rerun": "0"}}, "suite_name": "test_packet_publishing.py"}}}, "date": "August 02, 2024", "start_time": 1722589090.6338224, "total_suite": 1, "status": "FAIL", "status_list": {"pass": "0", "fail": "1", "skip": "0", "error": "0", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "1"}